src/main/java/seedu/address/logic/commands/FindByEmailCommand.java

```diff
@@ -0,0 +1,34 @@
+package seedu.address.logic.commands;
+
+import seedu.address.model.person.EmailContainsKeywordsPredicate;
+
+/**
+ * Finds and lists all persons in address book whose email contains any of the argument keywords.
+ * Keyword matching is case-insensitive.
+ */
+public class FindByEmailCommand extends AbstractFindCommand {
+
+    public static final String COMMAND_WORD = "find /e";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Finds all persons whose emails "
+            + "contain any of the specified keywords (case-insensitive) and displays them as a list with indices.\n"
+            + "Parameters: KEYWORD [MORE_KEYWORDS]...\n"
+            + "Example: " + COMMAND_WORD + " alex@gmail.com";
+
+    public FindByEmailCommand(EmailContainsKeywordsPredicate predicate) {
+        super(predicate);
+    }
+
+    @Override
+    public boolean equals(Object other) {
+        if (other == this) {
+            return true;
+        }
+
+        // instanceof handles nulls
+        if (!(other instanceof FindByEmailCommand otherFindCommand)) {
+            return false;
+        }
+        return super.predicate.equals(otherFindCommand.predicate);
+    }
```

Comment:
Perhaps can keep in view to use this.getPredicate() instead of directly accessing the super.predicate field although it is allowed (or remove the getPredicate() entirely, undecided as of now)